src/main/java/uwu/Ui.java

```diff
@@ -0,0 +1,123 @@
+package uwu;
+
+import uwu.task.Task;
+import uwu.task.TaskList;
+
+import java.util.Scanner;
```

Comment:
I noticed that in Storage.java import statements have different ordering (ie java.* is before uwu.*) Maybe you would like to standardize the ordering